app/src/test/java/com/kickstarter/viewmodels/DiscoveryFragmentViewModelTest.java

```diff
@@ -39,11 +39,12 @@ public void testProjectsEmitWithNewCategoryParams() {
     final TestSubscriber<Boolean> hasProjects = new TestSubscriber<>();
     vm.outputs.projects().map(ListUtils::nonEmpty).subscribe(hasProjects);
 
-    // Load initial params from activity.
+    // Load initial params and root categories from activity.
     vm.inputs.paramsFromActivity(DiscoveryParams.builder().sort(DiscoveryParams.Sort.HOME).build());
+    vm.inputs.rootCategories(CategoryFactory.rootCategories());
 
     // Should emit current fragment's projects.
-    hasProjects.assertValues(false, true);
```

Comment:
the projects no longer emit an initial empty array because there is no api delay with 2 requests: one for projects and one for categories